### DISCUSSION TITLE
Boundaries in CohesiveZoneAction

### DISCUSSION THREAD

**Discussion by ghost** [idaholab/moose#25690] [Q&A General] [answered]
Hello, 

Is there a way to set all of the interfacial boundary IDs (block0_block1 block1_block2 etc) as boundary parameter in the CohesiveZoneAction or do we have to name them all one by one ?

**Comment by lindsayad** [comment]
@jiangwen84 @arovinelli

**Comment by arovinelli** [comment, marked as answer]
@nshabib if you use the `split_interface=true` option (see [BreakMeshByBlockGenerator](https://mooseframework.inl.gov/source/meshgenerators/BreakMeshByBlockGenerator.html) ) yes, you have to list them all.

**Reply by ghost** [reply]
Oh ok, thank you!

**Reply by arovinelli** [reply]
@nshabib Just a clarification. 
You need to use `split_interface=true` only if the cohesive responses is different for different interfaces. 
If this is not the case, i.e. you use the same cohesive law with the same parameter for all interfaces, you can use `split_interface=false`. This will generate one single cohesive interface called `interface`

**Reply by ghost** [reply]
Yes I got that, thank you.
